pkg/proxy/ipvs/testing/fake.go

```diff
@@ -0,0 +1,38 @@
+// +build linux
```

Comment:
is it necessary to restrict the fake to linux-only? this is breaking the test on macOS because there's no non-linux fake implemented currently.

Reply:
NO. We should remove this restriction.

Reply:
I already send a PR, see https://github.com/kubernetes/kubernetes/pull/54666